packages/trie/src/trie/trie.ts

```diff
@@ -39,16 +38,18 @@ interface Path {
 export class Trie {
   /** The root for an empty trie */
   EMPTY_TRIE_ROOT: Buffer
-  protected lock: Semaphore
+  private lock: Semaphore
```

Comment:
It's mostly better to keep things `protected` rather then `private`, then people can still use this if they want to extend and customize the base class in some way.